.github/workflows/docker-compose-test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
   docker-compose-test:
     runs-on: ubuntu-latest
     timeout-minutes: 25
+    if: false  # Temporarily disabled due to Kubernetes auth configuration issues
```

Comment:
I'm ok with temporarily disabling this in this PR, but I'd like to have @masayag feedback to understand if we can remove it or if we need to re-enable it.